src/revChatGPT/V1.py

```diff
@@ -89,18 +89,25 @@ def __init__(
         session_client=None,
     ) -> None:
         self.config = config
-        self.session = session_client() if session_client else requests.Session()
+        self.session = (
+            session_client(trust_env=False) if session_client else requests.Session()
+        )
```

Comment:
Should trust environment. I tend to use that rather than putting it in config as it's faster